protocol/0080-SPOT-product_builtin_spot.md

```diff
@@ -0,0 +1,167 @@
+# Built-in [Product](./0051-PROD-product.md): Spot
+
+This built-in product provides spot contracts which allow the buying and selling of a "base" asset with a "quote" asset for immediate delivery.
+
+When trading Spot products, parties can only use assets they own - there is no leverage or margin.
+
+## 1. Product parameters
+
+1. `base_asset (Asset)`: this is used to specify the asset to be purchased or sold on the market.
+1. `quote_asset (Asset)`: this is used to specify the asset which can be exchanged for the base asset.
+
+## 2. Network Parameter
+
+1. `spot_trading_enabled`: parameter defines whether markets using Spot products are enabled on the network.
+
+## 3. Liquidity Monitoring parameters
+
+1. `time_window`: length of rolling window (in seconds) over which the maximum `total_stake` is measured.
+1. `target_stake_factor`: fraction of `total_stake` to be selected as the `target_stake`.
+
+## 4. Market parameters
+
+1. `market_decimal_places` should be used to specify the number of decimal places of the `quote_asset` when specifying order price.
+
+    The Cash Settled Futures spec could rename `market_decimal_places` to something more general, e.g. `price_decimal_places`.
+
+1. `position_decimal_places` should be used to specify the number of decimal places of the `base_asset` asset when specifying order size.
+
+    The Cash Settled Futures spec could rename `position_decimal_places` to something more general, e.g. `size_decimal_places`.
+
+## 5. Liquidity Commitments
+
+### Submissions
+
+A liquidity provision submitted to a `Spot` market allows a separate commitment amount to be specified for the buy and sell sides of the market. These commitment amounts are specified in the `quote_asset` and `base_asset` respectively.
+
+```psuedo
+Example `LiquidityProvisionSubmission` command to an ETH/DAI market:
+
+submission = {
+    "liquidityProvisionSubmission": 
+    {
+        marketId: "abcdefghiklkmnopqrstuvwxyz",
+        fee: "0.01",
+        buyCommitmentAmount: 15000
+        buys: [
+            {
+                offset: "1"
+                proportion: "1"
+                reference: "PEGGED_REFERENCE_BEST_BID"
+            }
+        ]
+        sellCommitmentAmount: 10
+        sells: [
+            {
+                offset: "1"
+                proportion: "1"
+                reference: "PEGGED_REFERENCE_BEST_ASK"
+            }
+        ]
+        reference: "example_liquidity_provision_submission"
+    }
+}
+```
+
+As the LP now has a different commitment amount on each side of the book, the following considerations must be made:
+
+- Physical Stake:
+  - An LPs `physical_stake` should be treated separately for each side of the book - call these the `buy_physical_stake` and the `sell_physical_stake`.
+  - The current `physical_stake` for market stake calculations is the smaller of the two values, where the `sell_physical_stake` is converted into the `quote_asset` at the current `mark_price`.
```

Comment:
"at the current mark_price" - does that mean the last traded price?

Reply:
Was thinking the `mark_price` should be calculated the same way as in a Futures market, so the `mark_price` wouldn't be the last traded price if `network.markPriceUpdateMaximumFrequency` is greater than `0s`.